### ISSUE TITLE
Document accessibility priority numbers better

### ISSUE THREAD

**Issue by jidanni** [htacg/tidy-html5#756]
On HTML Tidy for Linux version 5.2.0 (latest here on Debian) man page we read
```
   Diagnostics options:

       accessibility-check

              Type:    enum
              Default: 0 (Tidy Classic)
              Example: 0 (Tidy Classic), 1 (Priority 1 Checks), 2 (Priority 2 Checks), 3 (Priority 3 Checks)

              This option specifies what level of accessibility checking, if any, that Tidy should perform.

              Level 0 (Tidy Classic) is equivalent to Tidy Classic's accessibility checking.

              For more information on Tidy's accessibility checking, visit Tidy's Accessibility Page at http://www.html-
              tidy.org/accessibility/.
```
OK, but we may be offline. And that page is rather big. Therefore the
man page should also mention:
- What priority is "higher" 1 or 3?
- Does picking e.g., 3 include 1 and 2?

**Comment by ler762** [comment]
https://cygwin.com/acronyms/#PTC

The good news is that the next release has stalled, so if you come up with a patch it might make it into the next release.

The bad news is that the next release has stalled.  See  https://github.com/htacg/tidy-html5/issues/741

**Comment by geoffmcl** [comment]
@jidanni our documentation can always be improved... **always** ;=))

I guess the specific answer is 3 is higher than 1... with the default 0, being minimal accessibility checks, and I guess 3 would include 1, 2, etc... at least that is the way it looks in reading some of the code... 

See [access.h](https://github.com/htacg/tidy-html5/blob/next/src/access.h#L18)... with -

```C
 * The accessibility checks to perform depending on user's desire:
 *   1. priority 1
 *   2. priority 1 & 2
 *   3. priority 1, 2, & 3
```

And see `TY_(AccessibilityChecks)( doc )`, called at the end of `ParseDocument`... 

Maybe someone who understands this better could test and advise more...

I do not think there have been accessibility code since release `5.2`, Apr 7, 2016, in later releases, to current `next` branch, `5.7.16`. It remain unchanged... Back in 2015, I did lament in issue #388 that this code needed some TLC...

Look forward to doc improvements, patches, or PR... thanks...

**Comment by balthisar** [comment]
Fixed via 3dab4b31. Closing.
